### PR TITLE
Mark tracks as liked on stream view

### PR DESCRIPTION
**Pull request by Pitros**
Separated from #340

### CODE REVIEW

**Comment by weblancaster** [comment]
I get this error when I run your PR, can you take a look please.

screen shot 2015-09-17 at 11 28 10 am

**Comment by Pitros** [comment]
I'll take a look on that but for now don't merge that. I've got slightly better idea for that.

**Comment by weblancaster** [comment]
I won't merge.. great, thanks!

File: app/public/js/stream/streamCtrl.js [hunk 1]
```diff
@@ -36,4 +47,14 @@ app.controller('StreamCtrl', function ($scope, SCapiService, $rootScope) {
             });
     };
 
+    function markLikedTracks (tracks) {
+        var tracksData = tracks.collection || tracks;
+        for (var i = 0; i < tracksData.length; ++i) {
+            var track = tracksData[i].origin;
+
+            if (track.hasOwnProperty('user_favorite'))
+                track.user_favorite = ($scope.likes.indexOf(track.id) != -1);
```
Comment (weblancaster) [inline app/public/js/stream/streamCtrl.js]: @Pitros the bug here is that you are missing the key `collection` so should be `($scope.likes.collection.indexOf(track.id) != -1)`

if you think both cases are in need with and without the `collection` key you can do a shorthand if.
`$scope.likes.collection ? ($scope.likes.collection.indexOf(track.id) != -1) : ($scope.likes.indexOf(track.id) != -1)`

**Comment by weblancaster** [comment]
It's a small fix so I will fix @Pitros that's why I merged

**Pull request merged**